.circleci/config.yml

```diff
@@ -189,7 +189,7 @@ jobs:
 
   build-pi-strings:
     docker:
-      - image: circleci/rust:1.38
+      - image: karlekdocker/pippi:0.0.6
```

Comment:
We have our own image now :) How does karlekdocker/pippi differ from the default Rust image of CircleCI?

Reply:
Oh, is it the addition of `bash`?

Reply:
For instance it has `cmake` and all protobuf related tools. You can see here: https://github.com/lapsang-boys/pippi/blob/master/.circleci/images/primary/Dockerfile

Reply:
> Oh, is it the addition of `bash`?

No, both circleci/rust and karlekdocker/pippi has bash

Reply:
> For instance it has cmake and all protobuf related tools. You can see here: https://github.com/lapsang-boys/pippi/blob/master/.circleci/images/primary/Dockerfile

Thanks! Mayhaps you could give me the Docker one0one crash course later :)

Reply:
Actually, taking a look at the `Dockerfile` it is very clean and only contains things that are logical requirements of our build pipeline.

Reply:
> > For instance it has cmake and all protobuf related tools. You can see here: https://github.com/lapsang-boys/pippi/blob/master/.circleci/images/primary/Dockerfile
> 
> Thanks! Mayhaps you could give me the Docker one0one crash course later :)

Fo shizle